tests/unit_tests/pyanaconda_tests/modules/payloads/payload/test_module_payload_flatpak_manager.py

```diff
@@ -52,7 +56,7 @@
 REMOTE_PATH = "file:///flatpak/repo"
 
 
-class FlatpakManagerTestCase(unittest.TestCase):
+class FlatpakManagerTestCase:
```

Comment:
I removed unittest base class to allow the use of pytest parametrizer and fixtures. Since we are using pytest for invoking unittests and no setUp/tearDown methods are in use, plain classes should work just fine - and allow the use of pytest goodies.